### DISCUSSION TITLE
#32. VATZ biweekly meeting at 2023-04-26 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#427] [General]
### 1. Overall

<img width="1211" alt="image" src="https://user-images.githubusercontent.com/6308023/231610313-4bcdb6f7-6819-462a-b1cb-31ec5275d4c0.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
30 | 2/19 |  10
31 |  2/18 | 11
---

<img width="1383" alt="image" src="https://user-images.githubusercontent.com/6308023/234475284-f8e025ec-971e-4ec3-bd2e-7cc3e701841b.png">

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/323
- https://github.com/dsrvlabs/vatz/issues/415
- https://github.com/dsrvlabs/vatz/issues/388
   - https://github.com/dsrvlabs/vatz/pull/428
- https://github.com/dsrvlabs/vatz/issues/429
   - https://github.com/dsrvlabs/vatz/pull/430

**Comment by rootwarp** [comment]
# 준교
## Impl.
- https://github.com/dsrvlabs/vatz/issues/426

## Review
- https://github.com/dsrvlabs/vatz/pull/430

**Comment by meetrick** [comment]
## 황제
- no issue
- Todo
  - relayer vatz plugin
  - aptos vatz plugin

**Comment by kim201212** [comment]
# hyunggi

- https://github.com/dsrvlabs/vatz/issues/87#issuecomment-1514119048

**Comment by gnongs** [comment]
## Jinhong
- No issue.

**Comment by skonhwang** [comment]
## 승곤
- No issue

**Comment by hqueue** [comment]
## 형규

- Is this repo (github.com/dsrvlabs/vatz) available for external contributor?
-

**Comment by xellos00** [comment]
@xellos00 
plugin version
- install, start

----
